src/events/render-advert-label.spec.ts

```diff
@@ -45,75 +49,86 @@ describe('shouldRenderLabel', () => {
 		document.body.innerHTML = '';
 	});
 
+	const testCreativeTemplateId = 1234567;
+
 	it('renders an ad label for normal ads', async () => {
 		createAd(adverts['withLabel']);
-		return renderAdvertLabel(getAd()).then(() => {
+		return renderAdvertLabel(getAd(), testCreativeTemplateId).then(() => {
 			const ad = getAd();
-			expect(shouldRenderLabel(ad)).toBeTruthy();
+			expect(shouldRenderLabel(ad, testCreativeTemplateId)).toBeTruthy();
 		});
 	});
 
 	it('renders an ad label for interscroller ads', async () => {
 		createAd(adverts['interscroller']);
-		return renderAdvertLabel(getAd()).then(() => {
+		return renderAdvertLabel(getAd(), testCreativeTemplateId).then(() => {
```

Comment:
It would be more accurate to use the interscroller template ID for this test, since the test is specifically about the interscroller